modernism/static/_js/modernism.js

```diff
@@ -126,6 +126,24 @@ const setModalImage = (direction) => {
 
 }
 
+//Get the button
+let mybutton = document.getElementById("myBtn");
+// When scroll down 100px from the top of the document, show the button
+window.onscroll = function() {scrollFunction()};
+
+function scrollFunction() {
+  if (document.body.scrollTop > 200 || document.documentElement.scrollTop > 200) {
+    mybutton.style.display = "block";
+  } else {
+    mybutton.style.display = "none";
+  }
+}
+// When clicks on the button, scroll to the top of the document
+function topFunction() {
```

Comment:
Same here, no need to call it "Function", describe what happens if it is used.

Reply:
I changed it